agent/engine/errors.go

```diff
@@ -137,3 +125,133 @@ func (err TaskStoppedBeforePullBeginError) Error() string {
 func (TaskStoppedBeforePullBeginError) ErrorName() string {
 	return "TaskStoppedBeforePullBeginError"
 }
+
+// CannotStopContainerError indicates any error when trying to stop a container
+type CannotStopContainerError struct {
```

Comment:
If you wanted to be fancy, you could `go generate` the type definition, `Error` function, and `ErrorName` function since they're all the same.